src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -91,6 +91,7 @@
     <VSIXSourceItem Include="$(OutputPath)Microsoft.Extensions.FileSystemGlobbing.dll" />
     <VSIXSourceItem Include="$(OutputPath)Microsoft.Extensions.FileProviders.Abstractions.dll" />
     <VSIXSourceItem Include="$(OutputPath)Microsoft.Extensions.FileProviders.Physical.dll" />
+    <VSIXSourceItem Include="$(OutputPath)Microsoft.VisualStudio.LanguageServer.Protocol.Extensions.dll" />
```

Comment:
Should you also be including your own copy of Microsoft.VisualStudio.LanguageServer.Protocol.dll?  Seems like everyone else is (with a codebase).

Reply:
Probably should. Looks like I haven't directly used any types from that lib yet but better to add it now then try and track down the problem later 😄